python/doc/examples/functional_modeling/quick_start_functions.ipynb

```diff
@@ -0,0 +1,585 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The calling sequence of a vectorized Python function is:\n",
+    "\n",
+    "```\n",
+    "def mySimulator (x):\n",
+    "    [...]\n",
+    "    return y\n",
+    "\n",
+    "myfunction = PythonFunction(nbInputs, nbOutputs, func_sample = mySimulator)\n",
+    "```\n",
+    "\n",
+    "where \n",
+    "\n",
+    "* x: the input of the function, a `Sample` with size `nbExperiments` (`getSize`) and dimension `nbInputs` (`getDimension`),\n",
+    "* y: the output of the function\n",
+    "\n",
+    "  * a numpy `array` with `nbExperiments` rows and `nbOutputs` columns\n",
+    "  * or a `Sample` with size `nbExperiments` and dimension `nbOutputs`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following, we present an vectorization example based on the `numpy` module."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def mySimulatorVect (x):\n",
+    "    # Convert NumericalSample > Array Numpy\n",
+    "    x = np.array (x)\n",
+    "    x0 = x[: ,0] # Extract column 0\n",
+    "    x1 = x[: ,1]\n",
+    "    x2 = x[: ,2]\n",
+    "    y0 = x0 + x1 + x2\n",
+    "    y1 = x0 - x1 * x2\n",
+    "    # Stack the two rows\n",
+    "    y = np.vstack ((y0 ,y1 ))\n",
+    "    y = y.transpose ()\n",
+    "    return y"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "myfunctionVect = ot.PythonFunction (3, 2, func_sample = mySimulatorVect )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "outputVect =  ot.CompositeRandomVector(myfunctionVect, inputRandomVector)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[-0.00077008,-0.0165189]\n",
+      "[1.74231,1.40859]\n"
+     ]
+    }
+   ],
+   "source": [
+    "montecarlosize = 10000\n",
+    "outputSample = outputVect.getSample(montecarlosize)\n",
+    "\n",
+    "empiricalMean = outputSample.computeMean()\n",
+    "print(empiricalMean)\n",
+    "empiricalSd = outputSample.computeStandardDeviationPerComponent()\n",
+    "print(empiricalSd)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to manage the history\n",
+    "\n",
+    "The `MemoizeFunction` class defines an history system to store the calls to the function.\n",
```

Comment:
```suggestion
    "The `MemoizeFunction` class defines a history system to store the calls to the function.\n",
```
In modern English, "a history" is more frequently used than "an history" because the "h" sound is [strongly pronounced](https://www.alphadictionary.com/articles/drgw007.html).

Reply:
Ok. Fixed.